db/projects/komodo/project.yaml

```diff
@@ -0,0 +1,90 @@
+name: Komodo
+web:
+  - 'https://komodoplatform.com/latest-news/'
+start_date: '2014-01-01'
+description: 'Open-Source Infrastructure For An Interoperable Blockchain Ecosystem'
+ledgers:
+  - id: komodo
+    name: Komodo
+    type: blockchain
+    networks:
+      - id: main
```

Comment:
You can also add type of the network, main or test, so you would use
  type: main

Reply:
done